portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -19,6 +19,9 @@
 /** Comment data class */
 public class Comment {
 
+  /* Comment poster */
+  private String commentPoster;
```

Comment:
nit: commentPoster --> author

Reply:
Will address in this PR.

Reply:
Though I think `nickname` will be a better choice (still along the same lines).